macros/anaUltraScurve.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
 import os
```

Comment:
Running in python2.6.6 causes crash, maybe $PYTHONPATH is not set correctly in:

source $BUILD_HOME/cmsgemos/setup/paths.sh

However executing with python2.7 gives no problem.  What is the argument against requiring python2.7?  Installation recipe could just be updated include a local installation of python2.7 for all DAQ machines.